svir/utilities/utils.py

```diff
@@ -815,10 +816,28 @@ def get_style():
         'irmt/style_mode', style_mode_default, type=int)
     classes = settings.value(
         'irmt/style_classes', style_classes_default, type=int)
-
+    # look for the setting associated to the layer if available
+    force_restyling = None
+    if layer is not None:
+        # NOTE: I would use %s/%s instead of %s%s, but it does not work
+        value, found = QgsProject.instance().readEntry(
+            'irmt', '%s%s' % (layer.id(), 'force_restyling'))
+        if found:
+            force_restyling = True if value == 'True' else False
```

Comment:
The value I read from QgsProject is a string, and bool(value) would be always True unless the string is empty. So I prefer to keep this unchanged.